integration_test/validate_metadata_test.go

```diff
@@ -31,22 +30,19 @@ func TestValidateMetadataOfThirdPartyApps(t *testing.T) {
 	}
 }
 
-func TestMissingMetadataForThirdPartyApps(t *testing.T) {
+func TestRequireMetadataForAllThirdPartyApps(t *testing.T) {
 	parentDirectory := "third_party_apps_data/applications"
 	dirs, err := thirdPartyDataDir.ReadDir(parentDirectory)
-
 	if err != nil {
-		t.Error(err)
+		t.Fatal(err)
 	}
 
 	for _, dir := range dirs {
-		dirName := dir.Name()
-		dirPath := path.Join(parentDirectory, dirName)
-		if _, er := os.Stat(path.Join(dirPath, "metadata.yaml")); errors.Is(er, os.ErrNotExist) {
-			err = multierr.Append(err, er)
+		_, fileErr := os.Stat(path.Join(parentDirectory, dir.Name(), "metadata.yaml"))
+		if fileErr != nil {
```

Comment:
This can be:
```
		if _, fileErr := os.Stat(path.Join(parentDirectory, dir.Name(), "metadata.yaml")); fileErr != nil {
```
right?

Reply:
Changed in: [71711cb](https://github.com/GoogleCloudPlatform/ops-agent/pull/793/commits/71711cb9861cdafe5b2799b06df9aaa1f195cdcb)